CONTRIBUTING.md

```diff
@@ -1,297 +1,385 @@
+### Update Script
 
-Keep the package source files clean and remove obsolete or outdated code and unnecessary comments. Comment non-obvious code so that others can easily understand what it does.
+Update scripts are the PowerShell scripts responsible for getting information about the software's most current version, where it can be acquired from, and how to modify the source code to produce the package. It is typically called `update.ps1`.
+These scripts should be located in the root of the package directory, right next to the metadata (`.nuspec`) file.
+The update script should be compatible with PowerShell v5 and may not need to be compatible with PowerShell Core.
 
-# 3. Teamwork
+#### Use `UseBasicParsing` When Possible
 
-Good communication is essential so please take a look at [Code of Conduct](https://github.com/chocolatey/.github/blob/main/CODE_OF_CONDUCT.md) regarding it.
+If you use the PowerShell cmdlet `Invoke-WebRequest` in the update script, always make sure to also pass in `-UseBasicParsing` to this cmdlet. There may be cases where using this parameter is not possible. Add a comment before the call about why `-UseBasicParsing` can not be used in these cases.
+Any script that does not use this argument requires the Internet Explorer engine to be available and the web browser to have launched at least once before.
 
-### 3.1 Update CODEOWNERS
+#### Do Not Download Large Files
 
-If you are submitting or making changes to a package and want to ensure that you can review any further changes, add your name to the [CODEOWNERS](https://github.com/chocolatey-community/chocolatey-packages/blob/master/.github/CODEOWNERS) file.
+Unless a package installer/executable/archive needs some special handling (like reading the version from the file or something else not automatically handled by AU), do not download anything bigger than a few MB within `au_GetLatest` function. Usually, the files are downloaded during the updating process automatically or in the `au_BeforeUpdate` function.
+Try finding alternatives to reading the version from the file when needed to prevent the need to download the file if no version elsewhere is available, like checking the headers of the response before doing a download.
 
-### 3.2 Single package per pull request
+## FAQ
 
-All contributors should issue pull request containing single package. In special cases multiple packages per PR can be allowed.
+### Why Do We Automatically Close Issues?
 
-### 3.3 Open issues and pull request expiration
+The Chocolatey Packages repository is a shared repository that will be worked on by the entire Chocolatey Community. When an issue lives for a long enough time to become stale, it needs more interest in the Community for the issue to become fixed. As there needs to be more interest in the Community, it is unlikely the issue will be fixed anytime soon, and it will instead be automatically closed.
```

Comment:
```suggestion
The Chocolatey Packages repository is a shared repository that is worked on by the entire Chocolatey Community. When an issue lives for a long enough time to become stale, it needs more interest in the Community for the issue to become fixed. As there needs to be more interest in the Community, it is unlikely the issue will be fixed anytime soon, and it will instead be automatically closed.
```